src/lib/sessiondir.c

```diff
@@ -111,7 +111,13 @@ char *singularity_sessiondir_init(char *file) {
             singularity_message(DEBUG, "Cleanup thread waiting on child...\n");
 
             waitpid(child_pid, &tmpstatus, 0);
-            retval = WEXITSTATUS(tmpstatus);
```

Comment:
what does WEXITSTATUS mean? I only found [this](http://stackoverflow.com/questions/20465039/what-does-wexitstatusstatus-return) and it didn't tell me much.